.github/workflows/main.yml

```diff
@@ -0,0 +1,57 @@
+name: Lint and test
+
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  main:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Python 3.7
```

Comment:
Haven't we decided that our main Python version is 3.6?

Reply:
Aren't we going for the lowest possible version on the SDK?

Reply:
Yes, which is 3.6, in setup.py we have `python_requires=">=3.6"`

Reply:
Done. Please create a ticket to update to Python 3.7